### PR TITLE
HTML & CSS practice: Hooli-style Popup

### PR DESCRIPTION
**Pull request by UA-girl**
### HTML & CSS practice: Hooli-style Popup

[Demo](https://ua-girl.github.io/Popup-practice/) |
[Code base](https://github.com/UA-girl/kottans-frontend/tree/master/task_html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/UA-girl/task_html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/documents.png" alt="Documents">
+                                        <span>Documents</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner__label">
+                                    <input type="checkbox" class="checkbox checkbox__additional hidden"
+                                           id="popup-additional">
+                                    <label for="popup-additional" class="label_popup__additional">
```
Comment (A-Ostrovnyy) [inline submissions/UA-girl/task_html-css-popup/index.html:112]: If I understand you correctly you use BEM notation: block_element__modificator, and in this case, `.label_popup__additional` is class with modificator. In BEM methodology class with modificator can'texist on its own, he must be provided by original class without modificator. [Read more](https://ru.bem.info/methodology/quick-start/#%D0%9C%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80)

File: submissions/UA-girl/task_html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                </li>
+                <li class="nav-item nav-item__popup">
+                    <input type="checkbox" class="checkbox checkbox__main hidden" id="popup-main">
+                    <label for="popup-main" class="label_popup__main">
+                        <img class="checkbox-image checkbox-image__main top-nav-img" src="img/popup-button.png"
+                             alt="Open popup" width="25">
+                        <span class="hidden">Open popup menu</span>
+                    </label>
+
+                    <ul class="nav-list nav-list__inner">
```
Comment (A-Ostrovnyy) [inline submissions/UA-girl/task_html-css-popup/index.html:34]: Do you really need this `ul` with one `li` item?)

File: submissions/UA-girl/task_html-css-popup/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    transition: 0.3s ease;
+}
+
+.container {
+    box-sizing: border-box;
+    max-width: 70%;
+    margin: 0 auto;
+}
+
+.hidden {
```
Comment (A-Ostrovnyy) [inline submissions/UA-girl/task_html-css-popup/css/styles.css:36]: Watch the [video ](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkbox in the right way.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments above.

**Comment by A-Ostrovnyy** [comment]
Avoid styling by HTML tags. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Comment by UA-girl** [comment]
Thank you! Can you review again? ) I've tried to fix all ))))

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by UA-girl** [comment]
Thank you! ))

**Pull request merged**